README.rst

```diff
@@ -17,7 +17,7 @@ pfurl - v1.3.7
 Overview
 ********
 
-This repository provides a curl-like client tool that is also able to perform simple file/directory functions (such as zip/unzip) suitable for remote http transmission:
+This the bleeding edge repository that provides a curl-like client tool that is also able to perform simple file/directory functions (such as zip/unzip) suitable for remote http transmission:
```

Comment:
Grammar is off here.  Also I am guessing this line isn't valid now that it's moving back to master.

Reply:
Yep! Many thanks.

Reply:
Other than that... ok to merge as far as you can tell?